examples/ssh/cpu_usage_iostat.js

```diff
@@ -189,7 +212,14 @@ function get_status() {
         .then(calculateAverage)
```

Comment:
you shoudl calculate, then get start and end dates then truncate
if get start and end date is return error because the file doesn't exists just use console.warn to indicate that the doesn't exists and don't fail like the function truncate